content/en/docs/refguide/modeling/application-logic/workflows/boundary-events.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Boundary Events"
+url: /refguide/boundary-events/
+weight: 55
+---
+
+## Introduction
+
+Starting with Mendix version 10.15, the workflow editor has beta support for Non-Interrupting Timer boundary events.
+
+Examples:
+
+- Send a reminder when a task is not picked up after 2 days.
+- Escalate to a manager when a task is not completed after 10 days.
+
+## Overview
+
+In BPMN 2.0, boundary events are a type of event that is attached to the boundary of an activity (such as a task or subprocess) to handle exceptional situations or to trigger certain behaviours. The icon inside the event indicates the type of event (e.g. clock indicates it’s a timer event). Boundary events are always displayed by 2 circles (either solid or dashed) and are linked by a dotted line to the parent activity.
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/boundary-events/boundary-event.png" max-width=100% >}}
+
+A boundary event can be added to the following activities:
+
+- User Task
+- Multi-User Task
+- Wait for Notification
+- Call Workflow
+- Call Microflow
+
+A maximum of 5 boundary events can be added to an activity.
+
+Boundary events are intermediate events of an activity, which can be triggered while the related activity is in progress. When the boundary event is triggered its effect on the related activity depends on the boundary event type: Non-Interrupting or Interrupting boundary event.
```

Comment:
That's a term from BPMN, not sure we should mention it here. What do you think @PaulJongenMx ?

Reply:
If we don't add anything, I think it will be a bit confusing. Especially for new users or people who are unfamiliar with BPMN. Maybe we can refer to an external source?